internal/operatorcli/operator_cli.go

```diff
@@ -131,9 +151,14 @@ func RunRegister(c *cli.Context) error {
 		return cli.Exit(fmt.Sprintf("error creating logger %v", err), 1)
 	}
 
-	if passphrase == "" || keyStorePath == "" {
+	if passphrase != "" {
+		blsPassphrase = passphrase
+		ecdsaPassphrase = passphrase
+	}
+
+	if blsPassphrase == "" || ecdsaPassphrase == "" || keyStorePath == "" {
```

Comment:
with the new approach of having a passphrase per key, need to spilt this if into smaller and unrelated it statements